### PR TITLE
chore(textlint): move dependencies.

### PR DESCRIPTION
**Pull request by karak**
lerna is especially fat.

### CODE REVIEW

**Comment by karak** [comment]
Furthermore, package itself doesn't need lerna, as well as testing and building tools.

**Comment by azu** [comment]
> package itself doesn't need lerna

Yes. We can remove `lerna` from `gulp-textlint/package.json`

File: packages/gulp-textlint/package.json [hunk 1]
```diff
@@ -20,19 +20,19 @@
...
     "plugin-error": "^1.0.1",
-    "prettier": "^1.7.4",
     "textlint": "^11.2.1",
     "through2": "^2.0.0"
   },
   "devDependencies": {
+    "eslint": "^5.11.1",
+    "eslint-plugin-prettier": "^3.0.1",
     "gulp": "^4.0.0",
+    "lerna": "^2.8.0",
```
Comment (azu) [inline packages/gulp-textlint/package.json]: Would you remove it?

Comment (karak) [reply packages/gulp-textlint/package.json]: Sure.

**Comment by azu** [comment]
@karak Thanks!

**Pull request merged**